src/gdext/surface/pragmas.nim

```diff
@@ -0,0 +1,33 @@
+import gdext/core/userclass/propertyinfo
+
+type ExpEasingArgument* = enum
+  attenuation, positive_only
+type RangeArgument*  = enum
+  or_less, or_greater, exp, radians_as_degrees, degrees, hide_slider
```

Comment:
I had to move these here to avoid circular references. I find it ugly but I was too lazy to extract to a separate file used by both modules, which I'll do a bit later :)